doc/api/buffer.md

```diff
@@ -538,6 +538,17 @@ added:
 Creates and returns a new `Blob` containing a subset of this `Blob` objects
 data. The original `Blob` is not altered.
 
+#### `blob.bytes()`
```

Comment:
The entry for in which version this method was added is missing, do you mind adding it?

would be something like this:

```
<!-- YAML
added:
  - v20.16.0
-->
```